WalletWasabi/Blockchain/BlockFilters/IndexBuilderService.cs

```diff
@@ -111,7 +111,7 @@ public void Synchronize()
 					{
 						Interlocked.Exchange(ref _serviceStatus, Running);
 
-						SyncInfo syncInfo = null;
+						SyncInfo? syncInfo = null;
```

Comment:
This variable is never used out of the while loop. Instead of make it nullable (what we don't want to do) just declare it in the loop.

Reply:
Done